test/ansible/roles/build_images/tasks/main.yaml

```
@@ -0,0 +1,9 @@
+---
+- name: build images
+  shell: "export CONTAINER_TAG={{ container_tag }} && cd {{ src_dir }} && setsid make docker-build-all"
```

```suggestion
  shell: "setsid make docker-build-all"
  chdir: "{{ src_dir }}"
  environment:
    CONTAINER_TAG: "{{ container_tag }} "
```
might be more readable? (not sure about the chdir, could another parameter name)